warehouse/email/__init__.py

```diff
@@ -292,6 +292,15 @@ def send_email_verification_email(request, user_and_email):
     }
 
 
+@_email("new-email-added")
+def send_new_email_added_email(request, user_and_email):
+    user, email = user_and_email
+
+    return {
+        "email_address": email.email,
+    }
```

Comment:
Doesn't seem like we need to pass `user` here. (Also, it's generally preferred to pass individual arguments instead of tuples).

Reply:
It seems that `_send_email_to_user` expects a user: `TypeError: send_new_email_added_email() missing 1 required positional argument: 'user_or_users'`

Reply:
Ah right, the decorator. This definitely needs to not take a tuple in that case.

Reply:
Additional pointers would be appreciated 😅 to be honest, I referenced the test above

```
def send_email_verification_email(request, user_and_email):
    user, email = user_and_email
    ...
   ```

Reply:
Ah, nice catch, I was unaware of that. This seems fine then, sorry for the confusion!